### PR TITLE
Set up application structure for user client

### PR DESCRIPTION
**Pull request by grconnor**
[PT Link](https://www.pivotaltracker.com/story/show/175524235)
# This PR includes:
- React application.
- Cleaned files.
-----------------------------------------------------------------------------------------------------------------------------

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+          rel="noopener noreferrer"
+        >
+          Learn React
+        </a>
+      </header>
+    </div>
+  );
+}
+
+export default App;
```
Comment (jennymalmefeldt) [inline src/App.js]: Clean up this file

Comment (jennymalmefeldt) [reply src/App.js]: And rename the file to app.jsx

File: README.md [hunk 2]
```diff
@@ -1 +1,72 @@
-# transport_optimum_client
\ No newline at end of file
+# Transport Optimum
```
Comment (oliverochman) [inline README.md:1]: Clean this up, I would specify the purpose of this project here

File: public/manifest.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
+  "icons": [
```
Comment (oliverochman) [inline public/manifest.json]: you can remove the "icons" key here

File: src/App.css [hunk 4]
```diff
@@ -0,0 +1,38 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
```
Comment (oliverochman) [inline src/App.css]: please remove this file
scaffolded code

File: src/App.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
+import logo from './logo.svg';
+import './App.css';
+
+function App() {
```
Comment (oliverochman) [inline src/App.js]: clean this up, make sure you use the correct format

make this into a `jsx` file

File: src/App.js [hunk 6]
```diff
@@ -0,0 +1,25 @@
+import logo from './logo.svg';
```
Comment (oliverochman) [inline src/App.js]: no need for any of these imports

File: src/index.js [hunk 7]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
```
Comment (oliverochman) [inline src/index.js]: dont use scaffolded styling

**Review by oliverochman — CHANGES_REQUESTED** [review]

File: src/reportWebVitals.js [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+    import('web-vitals').then(({ getCLS, getFID, getFCP, getLCP, getTTFB }) => {
+      getCLS(onPerfEntry);
+      getFID(onPerfEntry);
+      getFCP(onPerfEntry);
+      getLCP(onPerfEntry);
+      getTTFB(onPerfEntry);
+    });
+  }
+};
+
```
Comment (johanperjulius1) [inline src/reportWebVitals.js:12]: What do we use this for? Is it for Jest or Enzyme?

File: package.json [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.11.4",
+    "@testing-library/react": "^11.1.0",
+    "@testing-library/user-event": "^12.1.10",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
+    "react-scripts": "4.0.0",
+    "web-vitals": "^0.2.4"
+  },
+  "scripts": {
```
Comment (johanperjulius1) [inline package.json:14]: Shouldn't we add the start-server script?

**Review by johanperjulius1 — CHANGES_REQUESTED** [review]

File: package.json (hunk 9 above)
Comment (oliverochman) [reply package.json:14]: you have it on line 15

Comment (grconnor) [reply package.json:14]: There is a start server script?

File: src/reportWebVitals.js (hunk 8 above)
Comment (oliverochman) [reply src/reportWebVitals.js:12]: https://create-react-app.dev/docs/measuring-performance/

Comment (grconnor) [reply src/reportWebVitals.js:12]: It was scaffolded with the react app, it's for performance

File: src/index.js (hunk 7 above)
Comment (grconnor) [reply src/index.js]: Removed

File: src/App.js (hunk 6 above)
Comment (grconnor) [reply src/App.js]: Removed

File: src/App.js (hunk 5 above)
Comment (grconnor) [reply src/App.js]: Cleaned up and suffix changed

File: src/App.css (hunk 4 above)
Comment (grconnor) [reply src/App.css]: Removed

File: package.json (hunk 9 above)
Comment (johanperjulius1) [reply package.json:14]: I meant the one for cypress so we can run the cypress command, but I realise thats another PR :)

File: src/reportWebVitals.js (hunk 8 above)
Comment (johanperjulius1) [reply src/reportWebVitals.js:12]: Okey, maybe I / we deleted it in other projects? :)

File: README.md (hunk 2 above)
Comment (grconnor) [reply README.md:1]: Done

**Review by oliverochman — APPROVED** [review]

**Pull request merged**